src/content/docs/support/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Products
+linkTitle: Products
+description: Commercial Products and Services for Kyverno
+type: docs
+---
+
+## Commercial Support Providers
+
+- **[Blakyaks](./support/blakyaks/)**  
+  Cloud-native, containers, serverless, automation.
+
+- **[Giantswarm](./support/giantswarm/)**  
+  Managed Kubernetes as a Service.
+
+- **[Infracloud](./support/infracloud/)**  
+  Kyverno Consulting and Implementation Partner.
+
+- **[KodeKloud](./support/kodekloud/)**  
+  Hands-on Kubernetes and DevOps Training.
+
+- **[Nirmata](./support/nirmata/)**  
+  Enterprise policy management and governance with Kyverno.
```

Comment:
Could you fix the links as well ? 🙏 This doesn't work currently, because the relative `support` folder doesn't exist (eg: `./niramata/` should work)